tests/integration/integration_test_case.py

```diff
@@ -50,16 +50,20 @@ def _setUpApp(self):
             'keys': {
                 EQ_USER_AUTHENTICATION_RRM_PRIVATE_KEY_KID: {'purpose': KEY_PURPOSE_AUTHENTICATION,
                                                              'type': 'private',
-                                                             'value': get_file_contents('third-party/sdc-rrm-authentication-signing-private-v1.pem')},
+                                                             'value': get_file_contents(
```

Comment:
I think all of these would be clearer with a new line after the `{`, something like:
```
EQ_USER_AUTHENTICATION_RRM_PRIVATE_KEY_KID: {
    'purpose': KEY_PURPOSE_AUTHENTICATION,
    'type': 'private',		     
    ....
}
```